script.js

```diff
@@ -1,4 +1,135 @@
+
+// Project Scissors, Paper Stone.....up to Part 2 Basic
+
+// generate function random number
+var randomNumEl = function(){
+  var randomEl = Math.random() * 3
+  var roundEl = Math.floor(randomEl)
+  var integer = roundEl + 1
+  return integer
+  }
+
+// global var
+var userWinEl = 0
+var programWinEl = 0
+var gameDrawEl = 0
+var totalCountEl = 0
+var currentMode = ``
+var userName = ``
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  if(currentMode == ``){
+    userName = input
+    currentMode = `spsGame`
+    myOutputValue = `Hi ${input}, welcome to the game. Now you can start your game by typing Scissors, Paper, Stone.<br>
+    if you want more fun, try our Reversed Mode by typing Reversed in front of selected object for more intesting result. Enjoy!`
+  } else if(currentMode == `spsGame`){
+// default error message
+var myOutputValue = `Error......   You type:  ${input}. Please check your typing and retry again`
+
+// generate function assign SPS
+var programEl = randomNumEl()
+if(programEl == 1){
+programEl = `Scissors`
+}
+if(programEl == 2){
+programEl = `Paper`
+}
+if(programEl == 3){
+programEl = `Stone`
+}
```

Comment:
You could actually implement this into your helper function too, so that it returns a string that is either scissors, paper or stone to the main function